worlds/aquaria/Locations.py

```diff
@@ -159,7 +159,7 @@ class AquariaLocations:
     }
 
     locations_simon = {
-        "Kelp forest, beating Simon says": 698156,
+        "Simon says area, beating Simon says": 698156,
         "Simon says area, Transturtle": 698216,
```

Comment:
```suggestion
        "Simon Says area, beating Simon Says": 698156,
        "Simon Says area, Transturtle": 698216,
```
Would recommend capitalizing "Says" here, since "Simon Says" is the name of a game, so it's grammatically correct to capitalize both words

Reply:
Here. I think that I found them all.